source/support.tex

```diff
@@ -2237,12 +2215,10 @@
 \tcode{operator delete}.
 
 \pnum
-\expects
 If an implementation has strict pointer safety\iref{basic.stc.dynamic.safety}
 then \tcode{ptr} is a safely-derived pointer.
 
 \pnum
-\expects
```

Comment:
Wouldn't it be a good idea to deindent the elements a little? Although it might not be worth it considering there aren't many examples like these:
![1593825063](https://user-images.githubusercontent.com/21071787/86502491-c1e22980-bd71-11ea-9dce-ceb21232a6e6.png)

Reply:
I think we do both styles. @jwakely , any opinion here?